src/core_codemods/django_session_cookie_secure_off.py

```diff
@@ -15,11 +15,21 @@
 
 class DjangoSessionCookieSecureOff(SemgrepCodemod, Codemod):
     METADATA = CodemodMetadata(
-        DESCRIPTION=("Sets Django's `SESSION_COOKIE_SECURE` flag if off or missing."),
+        DESCRIPTION=("Sets Django's `SESSION_COOKIE_SECURE` Flag if Off or Missing."),
```

Comment:
I think we need to keep the previous capitalization here (see comment about `DESCRIPTION` -> `CHANGE_DESCRIPTION` -> PR comment).